AdyenDropIn/Components/PaymentMethodListComponent.swift

```diff
@@ -38,14 +45,34 @@ internal final class PaymentMethodListComponent: ComponentLoader, PresentableCom
         self.style = style
     }
     
+    internal func reload(with components: [ComponentsSection]) {
+        componentSections = components
+        listViewController.reload(newSections: createListSections())
+    }
+    
+    internal func deleteComponent(at indexPath: IndexPath) {
+        componentSections.deleteItem(at: indexPath)
+        listViewController.deleteItem(at: indexPath)
+    }
+    
     // MARK: - View Controller
     
     /// :nodoc:
     internal var viewController: UIViewController { listViewController }
 
     private let brandProtectedComponents: Set = ["applepay"]
     
-    internal lazy var listViewController: ListViewController = {
+    internal lazy var listViewController: ListViewController = createListViewController()
+    
+    private func createListViewController() -> ListViewController {
```

Comment:
Why not just:

```
lazy var listViewController: ListViewController = { .. }()

Reply:
for reusability? :-)

Reply:
Do we reuse it?
It is a private function that is only used as for lazy loading 🤔

Reply:
it was :-D, I think its nicer this way

Reply:
as discussed offline :-)